v2/our-platform/api-reference/backtest-management.rst

```diff
@@ -0,0 +1,436 @@
+===================
+Backtest Management
+===================
+
+The QuantConnect.com API exposes methods for creating, reading, listing, updating, and deleting backtests in the user's account.
+
+|
+
+----------------------------------------------------------------
+
+Backtest management API calls return either a Backtest object, a Backtest List object, or a Backtest Report object.
+
+|
+
+Backtest Object
+---------------
+
+A Backtest objects describes a Backtest response packet from the QuantConnect.com API.
+
+Attributes
+==========
+
+.. list-table::
+   :header-rows: 1
+
+   * - Attribute
+     - Type
+     - Description
+   * - name
+     - string
+     - Name of the backtest.
+   * - note
+     - string
+     - Note on the backtest, attached by the user.
+   * - backtestId
+     - string
+     - Assigned backtest Id.
+   * - completed
+     - boolean
+     - True when the backtest is completed.
+   * - progress
+     - decimal
+     - Progress of the backtest in percent 0-1.
+   * - result
+     - A BacktestResult object (see full API reference for its attributes)
```

Comment:
Please add this object.

Reply:
BacktestResult has a large, 19-class dependency chain, a big part of why the YAML grew to 2300 lines.

If we add BacktestResult, this RST file will also need to add:
- AlgorithmPerformance
  - TradeStatistics
  - PortfolioStatistics
  - Trade
    - TradeDirection
- AlphaRuntimeStatistics
   - InsightScore
- Chart
   - Series
    - ChartPoint
- Order
   - Symbol
   - SecurityType
   - OrderDirection
   - OrderSubmissionData
- OrderEvent
   - OrderStatus
   - OrderFee
   - CashAmount

Reply:
Correct. The API should be fully documented. Please add the needed objects.